contracts/contracts/token/OUSD.sol

```diff
@@ -270,9 +270,13 @@ contract OUSD is Governable {
         uint256 _value
     ) external returns (bool) {
         require(_to != address(0), "Transfer to zero address");
-        require(_value <= allowances[_from][msg.sender], "Allowance exceeded");
+        uint256 userAllowance = allowances[_from][msg.sender];
+        require(_value <= userAllowance, "Allowance exceeded");
 
-        allowances[_from][msg.sender] -= _value;
+
```

Comment:
⚪️ Don't think this needs two whitespace lines?

Reply:
Agreed fixed